query/src/main/java/io/keen/client/java/AbsoluteTimeframe.java

```diff
@@ -46,16 +46,20 @@ public AbsoluteTimeframe(String start, String end) {
      */
     @Override
     public Map<String, Object> constructTimeframeArgs() {
-        Map timeframe = null;
+        Map<String, Object> timeframe = null;
+
         if (start != null && end != null) {
-            Map absoluteTimeframe = new HashMap<String, Object>();
+            Map<String, Object> absoluteTimeframe = new HashMap<String, Object>(3);
```

Comment:
Yeah I've been meaning to ask about why we keep cross-compiling for 1.6 on all platforms across the board. We should have a deeper discussion about that, because this is just one of the ugly little retro things we have to keep dealing with if targeting 1.6.